apps/frontend/src/pages/LandingPage.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Box, Button, Image, VStack } from '@chakra-ui/react';
+
+function AuthButtons() {
+  /* Temporary styles for buttons; to be replaced with Chakra 'useStyleConfig'. */
+  const buttonStyles = {
+    height: '4em',
+    width: { base: '50%', md: '50%' },
+    fontSize: { base: 'lg', md: 'xl' },
+  };
+
+  return (
+    <VStack w={'100%'} spacing={4}>
+      <Link to="/signup">
```

Comment:
What's the difference between `signup` and `register` (unless `signup` is actually the `login` page; if so, perhaps we should rename these pages?)